alpine/version.txt

```diff
@@ -1 +1 @@
-3.21.3-2
\ No newline at end of file
+3.21.3-3
```

Comment:
I'm 99% sure this is not needed. We don't do anything in the alpine image except add labels. This would only be needed if we upgraded existing packages or installed new packages. https://github.com/astronomer/ap-vendor/blob/main/alpine/Dockerfile

Reply:
```suggestion
3.21.3-2
```